Tiltfile

```diff
@@ -252,18 +253,37 @@ def build_docker_image(image, context, binary_name, additional_docker_build_comm
 
     # Set up an image build for the provider. The live update configuration syncs the output from the local_resource
     # build into the container.
-    docker_build(
-        ref = image,
-        context = context + "/.tiltbuild/bin/",
-        dockerfile_contents = dockerfile_contents,
-        build_args = {"binary_name": binary_name},
-        target = "tilt",
-        only = binary_name,
-        live_update = [
-            sync(context + "/.tiltbuild/bin/" + binary_name, "/" + binary_name),
-            run("sh /restart.sh"),
-        ],
-    )
+    if settings.get("build_engine") == "podman":
+        bin_context = context + "/.tiltbuild/bin/"
+        local("tee %s/Dockerfile" % (shlex.quote(bin_context)), quiet = True, stdin = dockerfile_contents)
```

Comment:
nit
can we use a subfolder or a different file name for each provider?

Reply:
The context is already different for each provider, so `context + "/.tiltbuild/bin/" + "Dockerfile"` is unique per provider.

Reply:
ack